README.md

```diff
@@ -11,10 +11,14 @@ Humble | Garden | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | on
 Iron | Fortress | [iron](https://github.com/gazebosim/ros_gz/tree/iron) | https://packages.ros.org
 Iron | Garden | [iron](https://github.com/gazebosim/ros_gz/tree/iron) | only from source
 Iron | Harmonic | [iron](https://github.com/gazebosim/ros_gz/tree/iron) | only from source
+Jazzy* | Fortress | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | https://packages.ros.org
+Jazzy* | Garden | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | only from source
+Jazzy* | Harmonic | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | only from source
```

Comment:
I wasn't opposed to adding Jazzy, just that where we put "only from source" needs to change since it's what we expect the future state to be. 

```suggestion
Jazzy* | Fortress | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | only from source
Jazzy* | Garden | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | only from source
Jazzy* | Harmonic | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | [only from source](https://packages.ros.org)
```